python/tools/audioview_se.py

```diff
@@ -323,15 +383,25 @@ def main(args):
     argParser.add_argument(
         "-w",
         "--tty",
-        default = "/dev/tty.usbmodem1234561", # "/dev/tty.usbmodem1234561"
+        default = "COM4", # "/dev/tty.usbmodem1234561"
```

Comment:
Did you mean to leave this pointing to COM4?

Reply:
Thank you. Fixed